### DISCUSSION TITLE
合約更新

### DISCUSSION THREAD

**Discussion by MartinYeung5** [openbuildxyz/mina-zkapp-bootcamp#411] [Q&A]
有沒有人可以講解這個部分是怎樣做?

**Comment by logere** [comment]
更新合约代码吗

**Reply by logere** [reply]
可以参考这里更新 zkapp 账户的 VerificationKey https://docs.minaprotocol.com/zkapps/writing-a-zkapp/feature-overview/upgradability#upgrading-a-zkapp

**Reply by MartinYeung5** [reply]
是啊, 謝謝, 
我去看看
